locale/loris.pot

```diff
@@ -156,6 +162,12 @@ msgstr ""
 msgid "Date of Birth"
 msgstr ""
 
+msgid "Visits"
+msgstr ""
+
+msgid "Cohorts"
```

Comment:
since we have already `Cohort` in singular, I would move the plural version just right after the singular one.